app/controllers/admin/editions_controller.rb

```diff
@@ -426,6 +427,12 @@ def remove_blank_parameters
     params.reject! { |_, value| value.blank? }
   end
 
+  def infer_blank_checkbox_params
+    return if edition_params.empty? || !preview_design_system_user?
+
+    edition_params[:political] = "0" if edition_params[:political].blank?
+  end
```

Comment:
> The checkbox component does not pass a value via a hidden field when unchecked.

Could this actually be due to `remove_blank_parameters` which runs before anything else?

https://github.com/alphagov/whitehall/blob/d10c50548b3bca46df84d8434a56a26b10464d77/app/controllers/admin/editions_controller.rb#L426-L428

Perhaps a cleaner approach would be to avoid removing unchecked checkboxes in the first place, rather than removing them (`remove_blank_parameters`) and then adding them back in again (`infer_blank_checkbox_params`).

I guess something like this would probably do the trick, although there might be a nicer way to do it:

```ruby
def remove_blank_parameters
  checkbox_fields = %i[political]
  params.reject! { |key, value| value.blank? && !checkbox_fields.include?(key) }
end
```